scripts/notify-slack.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+curl \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '
+
+  {
+    "blocks": [
+      {
+        "type": "context",
+        "elements": [
+          {
+            "type": "mrkdwn",
+            "text": ":rocket: *Preview Deployed*"
+          }
+        ]
+      },
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+            "text": "> '"$COMMIT_MSG"'"
+        }
+      },
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": "<https://github.com/okta/odyssey/pull/'"$PULL_REQUEST_ID"'|Pull Request #'"$PULL_REQUEST_ID"'> ∙ *<'"$PREVIEW_URL"'|View Preview>*"
```

Comment:
Let's try to condense this:

```suggestion
          "text": "<https://github.com/okta/odyssey/pull/'"$PULL_REQUEST_ID"'|PR #'"$PULL_REQUEST_ID"'> ∙ *<'"$PREVIEW_URL"'|View Preview>*"
```